lib/backend/backend.go

```diff
@@ -244,6 +245,14 @@ func NextPaginationKey(r types.Resource) string {
 	return string(nextKey([]byte(r.GetName())))
 }
 
+// MaskKeyName masks the given key name.
+// e.g "123456789" -> "******789"
+func MaskKeyName(str string) string {
+	hiddenBefore := int(math.Floor(0.75 * float64(len(str))))
```

Comment:
You don't need to import math to accomplish this.
```suggestion
	hiddenBefore := int(0.75 * float64(len(str)))
```